pkg/operator/controller/vulnerabilityreport.go

```diff
@@ -242,6 +245,14 @@ func (r *VulnerabilityReportReconciler) submitScanJob(ctx context.Context, owner
 		WithCredentials(credentials).
 		Get()
 
+	if err != nil {
+		if errors2.Is(err, kube.ErrNodeNameNotFound) {
+			log.V(1).Info("Cannot run a scan job as nodeName is not available")
```

Comment:
We should explicitly state in logs that we're ignoring a workload scan because there are no pods (= 0 replicas). "nodeName is not available" is pretty scary and may mislead to infrastructure problems rather than deliberate logic implemented by Starboard.